health-and-fitness-organizer/src/pages/Homepage.js

```diff
@@ -0,0 +1,47 @@
+import Carousel from "react-bootstrap/Carousel";
+import CardHome from "../UI/CardHome";
+import workoutImg from "../Assests/images/workout.jpg";
+import healthContentImg from "../Assests/images/health-content.jpg";
+import styles from "./Homepage.module.css";
+import { bgcolor } from "@mui/system";
+
+const Homepage = () => {
+  const page = [
+    {
+      id: 1,
+      title: "Workout Plans",
+      description:
+        "Managing and creating weekly workout plans for your convenience",
+      image: workoutImg,
+    },
+    {
+      id: 2,
+      title: "Health Content",
+      description: "Managing and creating health content for you routine",
+      image: healthContentImg,
+    },
+  ];
+
+  return (
+    <div className={styles.homepage}>
+      <Carousel variant="dark" interval={null} indicators={null}>
```

Comment:
just wondering, are the null values necessary?  if you just didn't include the props would it do the same thing?

Reply:
yes, the default in Bootstrap is not null